src/core/include/core.h

```diff
@@ -16,7 +16,9 @@ class Core {
 		return instance;
 	}
 	void LoadHeatmap(std::string file_path_mv) {}
-	void LoadGeometry(std::string trm_file_path_mv, std::string t2d_file_path_mv);
+	void LoadGeometry(std::string trm_file_path_mv, 
+									  std::string t2d_file_path_mv,
```

Comment:
Do not use tabs for aligning. Thery are only for indent, that is level 1 here. The rest should be spaces